tests/unittests/utils/test_ultralytics.py

```
@@ -0,0 +1,836 @@
+        def fake_predict(img, **kwargs):
+            predict_calls.append(img)
+            return [_Result()]
+
+        model = self._make_model(monkeypatch, predict=fake_predict)
+
+        prompt = fol.Detections(
+            detections=[fol.Detection(label="x", bounding_box=[0, 0, 1, 1])]
+        )
+        original_predictor = model._model.predictor
+
+        with pytest.raises(ValueError):
```

Ruff (0.15.12)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)